project/Settings.scala

```diff
@@ -5,7 +5,7 @@ import sbtassembly.AssemblyKeys._
 object Settings {
   val buildName = "hazelcast-spark"
   val buildVersion = "0.1-SNAPSHOT"
-  val buildScalaVersion = "2.10.5"
+  val buildScalaVersion = "2.11.8"
```

Comment:
You can rewrite your commit history with interactive rebase in git using `git rebase -i master` on your branch. You can have a look at git rebase tutorials on the web.